sql/moz-fx-data-shared-prod/google_ads_derived/conversion_event_categorization_v1/query.sql

```diff
@@ -3,19 +3,26 @@
 --Note: Max cohort date cannot be more than 7 days ago (to ensure we always have at least 7 days of data)
 WITH clients_first_seen_14_days_ago AS (
   SELECT
-    client_id,
-    first_seen_date,
-    country,
-    attribution_campaign,
-    attribution_content,
-    attribution_dltoken,
-    attribution_medium,
-    attribution_source
+    cfs.client_id,
+    cfs.first_seen_date,
+    m.first_seen_date AS first_main_ping_date,
+    cfs.country,
+    cfs.attribution_campaign,
+    cfs.attribution_content,
+    cfs.attribution_dltoken,
+    cfs.attribution_medium,
+    cfs.attribution_source
   FROM
-    `moz-fx-data-shared-prod.telemetry.clients_first_seen` --contains all new clients, including those that never sent a main ping
+    `moz-fx-data-shared-prod.telemetry.clients_first_seen` cfs --contains all new clients, including those that never sent a main ping
+  LEFT JOIN
+    `moz-fx-data-shared-prod.telemetry_derived.clients_first_seen_v1` m
+    ON cfs.client_id = m.client_id
+    AND m.first_seen_date
+    BETWEEN DATE_SUB(cfs.first_seen_date, INTERVAL 1 DAY)
```

Comment:
```suggestion
    -- join so that we only get "first main ping" dates from clients that sent their first main ping within -1 and +6 days from their first_seen_date.
    -- we will miss ~5% of clients that send their first main ping later, this is a trade-off we make to have a two-week reporting cadence (one week to send their first main ping, then we report on the outcomes *one week after that*
    BETWEEN DATE_SUB(cfs.first_seen_date, INTERVAL 1 DAY)
```